### PR TITLE
Upgrade to new tc-platform with no leak on passive for m&M

### PR DESCRIPTION
**Pull request by mathieucarbou**
Also brings new way of supported management for plugins and services

See: https://github.com/Terracotta-OSS/terracotta-platform/pull/311

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -51,7 +51,7 @@ ext {
   ehcache2Version = '2.10.3'
 
   // Clustered
-  terracottaPlatformVersion = '5.3.0-pre9'
+  terracottaPlatformVersion = '5.3-SNAPSHOT'
```
Comment (mathieucarbou) [inline build.gradle]: to be changed

File: build.gradle [hunk 2]
```diff
@@ -124,6 +124,7 @@ subprojects {
     }
     mavenCentral()
     maven { url "http://repo.terracotta.org/maven2" }
+    mavenLocal()
```
Comment (mathieucarbou) [inline build.gradle]: to be removed

**Comment by anthonydahanne** [comment]
test this please

**Comment by anthonydahanne** [comment]
test this please

**Comment by mathieucarbou** [comment]
build stuck ?

**Comment by anthonydahanne** [comment]
@mathieucarbou yep; @ljacomet told me he raised a ticket at cloudbees

**Comment by ljacomet** [comment]
test this please

**Comment by ljacomet** [comment]
Build fails with a number of test failures due to NPE

**Comment by anthonydahanne** [comment]
test this please
weird test failure : org.ehcache.impl.internal.executor.PooledExecutionServiceTest.testAllThreadsAreStopped

**Comment by anthonydahanne** [comment]
tada ! @ljacomet

**Pull request merged**